demo/config/routes.rb

```diff
@@ -12,6 +12,7 @@
   scope path: Rails.env.production? ? "/view-components" : "/" do
     get "/auto_complete", to: "auto_complete_test#index"
     resources :toggle_switch, only: [:create]
+    resources :nav_list_items, only: [:index]
```

Comment:
I had trouble hooking up these routes for production, if you goto the live site now https://primer.style/view-components/lookbook/ the controller actions don't work. It's cause they'd need passthroughs from the DNS magic we have. But I thought maybe we could fix it here by changing the scope for them to `/view-components/lookbook-controllers/` ? Not blocking this pr, just be aware for when it merges.

Reply:
Oh that's really interesting. Is it as simple as adding another scope around these `resources` calls?